soc/arm/nordic_nrf/nrf54l/Kconfig.soc

```diff
@@ -0,0 +1,56 @@
+# Nordic Semiconductor nRF54 MCU line
+
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_NRF54LX
+
+config SOC_NRF54L15
+	bool "NRF54L15"
+
+config SOC_NRF54L15_ENGA
+	select SOC_NRF54L15
+	bool "NRF54L15 ENGA"
+
+config SOC_NRF54L15_ENGA_CPUAPP
+	select ARM
+	select ARMV8_M_DSP
+	select CPU_CORTEX_M33
+	select CPU_HAS_ARM_MPU
+	select CPU_HAS_ICACHE
+	select CPU_HAS_ARM_SAU
+	select CPU_HAS_FPU
+	select HAS_HW_NRF_RADIO_IEEE802154
+	select HAS_POWEROFF
+	select SOC_NRF54L15_ENGA
+	bool "NRF54L15 ENGA CPUAPP"
+
+config SOC_NRF54L_VREG_MAIN_DCDC
+	bool "Enable nRF54L DC/DC converter."
+	help
+	  To enable, an inductor must be connected to the DC/DC converter pin.
+
+config SOC_NRF54L_NORMAL_VOLTAGE_MODE
+	bool "Enable nRF54L Normal Voltage Mode."
+
+config SOC_NRF54L_GLITCHDET_WORKAROUND
+	bool "Enable workaround that disables glitch detector"
```

Comment:
can we set it as EXPERIMENTAL to make sure it is either highlighted to integrator (or that option is removed)?

Reply:
Also remove Enable when touching the file anyway.